### PR TITLE
LPS-198302 - Fix Workflow | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** [LPS-198302](https://liferay.atlassian.net/browse/LPS-198302)
**Tests:** 

- MyWorkflowTasks#AccountUserCanApproveTask
- MyWorkflowTasks#TaskIsNotDisplayedAfterOrderIsDeleted

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 1]
```diff
@@ -137,12 +137,12 @@ definition {
...
 		Workflow.assignToMeTaskByActions(
 			workflowAssetTitle = "Order",
-			workflowAssetType = "Commerce Order",
+			workflowAssetType = "Order",
 			workflowTask = "Review");
 
 		Workflow.approveTaskByActions(
 			workflowAssetTitle = "Order",
-			workflowAssetType = "Commerce Order",
+			workflowAssetType = "Order",
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:145]: The assignee has been changed.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 2]
```diff
@@ -1693,7 +1691,7 @@ definition {
 
 		Workflow.viewTaskAssignedToMyRoles(
 			workflowAssetTitle = "Order",
-			workflowAssetType = "Commerce Order",
+			workflowAssetType = "Order",
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1694]: The assignee has been changed.

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase [hunk 3]
```diff
@@ -1683,8 +1683,6 @@ definition {
 
 		CommerceNavigator.gotoMiniCart();
 
-		Button.click(button = "Review Order");
-
```
Comment (thaiscabrals) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase:1687]: Behavior has changed, there is no Review Order anymore.

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hey @thaiscabrals, did we find the ticket that caused this change?

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**